### PR TITLE
Fix nodejs images

### PR DESCRIPTION
**Pull request by goldmann**
We need to install required libraries and make sure the working
directory is writeable.

### CODE REVIEW

File: docker/runtimes/action-nodejs-6/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,69 @@ RUN curl -L -o nodejs.tar.gz https://github.com/apache/incubator-openwhisk-runti
   && rm -rf /opt/app-root/src/nodejs/ /opt/app-root/src/nodejs.tar.gz \
   # Install
   && npm install \
+  && npm install \
```
Comment (goldmann) [inline docker/runtimes/action-nodejs-6/Dockerfile:15]: This is ugly, but I don't know now how to do it better.

Comment (jcrossley3) [reply docker/runtimes/action-nodejs-6/Dockerfile:15]: So running the thing twice fixes the perms issues?

File: docker/runtimes/action-nodejs-6/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,69 @@ RUN curl -L -o nodejs.tar.gz https://github.com/apache/incubator-openwhisk-runti
...
+    winston@2.3.0 \
+    ws@1.1.1 \
+    xml2js@0.4.17 \
+    xmlhttprequest@1.8.0 \
+    yauzl@2.7.0 \
   && npm cache clean --force
 
+USER root
+
+RUN chown -R default:root /opt/app-root/src \
```
Comment (jcrossley3) [inline docker/runtimes/action-nodejs-6/Dockerfile:74]: Ah, here we fix perms issues!

File: docker/runtimes/action-nodejs-6/Dockerfile (hunk 1 above)
Comment (goldmann) [reply docker/runtimes/action-nodejs-6/Dockerfile:15]: Haha, good catch, fixing :)

Comment (goldmann) [reply docker/runtimes/action-nodejs-6/Dockerfile:15]: Uhm, no, actually this is how it should be.

File: docker/runtimes/action-nodejs-6/Dockerfile (hunk 2 above)
Comment (goldmann) [reply docker/runtimes/action-nodejs-6/Dockerfile:74]: Yah, I assumed that s2i image has already the correct permissions set but it's not the case.

**Pull request merged**